docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
My local `compose` was working fine, other than starting WEBrick instead of Unicorn (which, locally, isn't a big deal). 

I'm mostly just concerned to make sure this won't break EB or Circle if we were to push it up. If it doesn't actually run Unicorn, that's not a big deal at the moment.